environment-dev.yml

```diff
@@ -4,12 +4,11 @@ channels:
 dependencies:
   - pandas==1.5.2
   - numpy>=1.18.5
-  - pyarrow>=4.0.1
-  - ray-default>=1.4.0
+  - ray-default>=1.13.0
+  - pyarrow
```

Comment:
why remove minimal requirement on pyarrow?

Reply:
I'm not sure. At the moment, we can keep at least the lower bounds of dependency versions for different engines consistent. However, with each new engine added, this will become more and more difficult. Perhaps at some point, some engines will become completely incompatible...